index.rst

```diff
@@ -52,6 +52,64 @@
 
 .. Add content here.
 .. Do not include the document title (it's automatically added from metadata.yaml).
+.. _create-your-own-technote:
+
+Create Your Own Technote
+========================
+
+Creating your own technote just like this one is simple. Here we will explore how to create one.
+
+1. First thing to do is create your repository. Go to Slack and message **create project** to **sqrbot-jr**. You will then answer a series of questions. For the type of Documentation do reStructuredText. For Technote title and abstract or description do whatever. For Series TSTN. For Initial Copyright Holder do Association of Universities for Research in Astronomy. 
+
+.. note::
+
+   Before continuing with the steps check out the github repo you just created. It has a readme with lots of useful information such as a link to where your website is hosted. 
+
+2. Perfect, your boilerplate repo is created. Pull it down somewhere on your local machine.
+
+.. prompt:: bash
+
+   git clone https://github.com/lsst-tstn/tstn-005.git
```

Comment:
```
git clone https://github.com/lsst-tstn/tstn-{series_number}.git
```

You can find this information at the end of sqrbot-jr's successful creation of the repo.